testdata/simple/5a75f537726934e4b62833ab6d5dca41/tests/2022-07-23T07-15-34.848Z!2022-07-23T07-15-34.869Z

```diff
@@ -0,0 +1,11 @@
+2022-07-23T07:15:34.848Z First Test Log Line
+2022-07-23T07:15:34.866Z [js_test:geo_max:CheckReplOplogs] New session started with sessionID: {  "id" : UUID("4983fd5c-898a-4435-8523-2aef47ce91f3") } and options: {  "causalConsistency" : false }
+2022-07-23T07:15:34.868Z [js_test:geo_max:CheckReplOplogs] Recreating replica set from config {
+2022-07-23T07:15:34.868Z [js_test:geo_max:CheckReplOplogs] \t"_id" : "rs",
+2022-07-23T07:15:34.868Z [js_test:geo_max:CheckReplOplogs] \t"version" : 5,
```

Comment:
maybe it would be more straightforward to save the log lines as a json 2d array? Resmoke sends them to us this way, so we could literally just proxy them to S3. 
I could also picture it would make it easier when we parse it on retrieval if the timestamp and log line are already broken down for us.

Reply:
I don't know if it makes a huge difference... It's marginally easier to stream an array of lines or a jsonl file than a single json array object. I do _like_ the idea of using a standard format here though.

Reply:
I guess I'd vote for either jsonl (one json array per line) or be fine with just doing a big json array and relying on a streaming JSON reader. Do you have preferences?

Reply:
Updated to use jsonl. Added extensions to make life easier for future folks downloading and using these files and for editing locally.